android-lib/src/androidTest/java/com/kinvey/androidTest/store/datastore/DataStoreMultiInsertTestOrderKotlin.kt

```diff
@@ -16,7 +16,6 @@ import org.junit.Ignore
 
 @RunWith(AndroidJUnit4::class)
 @SmallTest
-@Ignore("Ignored tests in Kotlin for now while Kotlin migration is not done")
```

Comment:
why the tests are failing? the migration should not be a reason why we are ignoring tests

Reply:
Locally this tests always worked. Found that name of the test file was different that test class name, should fix issue.